Libs/Gtk-3.0/Classes/TextView.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using GObject.Native;
+
+namespace Gtk
+{
+    public partial class TextView
+    {
+        public static TextView New()
+            => ObjectWrapper.WrapHandle<TextView>(Native.TextView.Instance.Methods.New(), false);
```

Comment:
The wrapper checks if an object is already known for this pointer and afterwards calls the constructor per reflection. But as we are creating a new instance we can omit this check as we now there is no other instance and direcly call a constructor.